src/components/Books/BooksList.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
```

Comment:
- [ ] The structure of your project files **does not follow**  _the ducks pattern_.  Kindly ensure that your project files are structured conforming to _the duck pattern_, as below:

```
 ...
    /src
        |---/components
        |---/redux
            |--- /books
                    | books.js
            |--- /categories
                    | categories.js
            | cofigureStore.js
        | index.js
```

Reply:
Hi @rloterh ,
Thank you for your valuable feedback,
I'm just confusing about my project structure. I find the mine meet the duck pattern. Please give more details about what I should change exactly. 
Looking for your feedback.
Thank you !
![bookstore_structure](https://user-images.githubusercontent.com/85107514/144895205-edd7c731-5688-4c3e-8386-ec7a5f371cd5.png)

Reply:
**Hi @Taher-web-dev** 
You are right about this. I apologize I missed this important step during the first review. Kindly note that this was not a deliberate attempt.
Cheers, and happy coding! 😄